### PR TITLE
adding a gallery test page and draft for Image Magick

### PR DESCRIPTION
**Pull request by arrowarchive**
I added all the thumbnails and a page for them to be displayed. I also added a draft of my tutorial for Image Magick

### CODE REVIEW

File: docs/web/galleryTest.html [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+        <link rel="stylesheet" type="text/css" href="experimental.css"/>
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <meta charset="utf-8"/>
+    </head>
+    <body>
+        <div class="grid">
+            <h1>Image Testing</h1>
+            <p>This page is meant to test image display. I rendered all the thumbnails, but on the main archive, they aren't displaying right. The width is always 200 px with landscapes, while the height is 200 px for portraits.</p>
+            <p>I have a feeling that when using the thumbnail property, it tried to keep the aspect ratio of the thumbnails. If that is true, I will need help recoding the gallery. They aren't aligned evenly, and the images hover over the overlays or have them protrude from the sides. Let's see if my hypothesis is right.</p>
+            <p>-Rachel</p>
```
Comment (am0eba-byte) [inline docs/web/galleryTest.html:14]: I'll be interested to see how the Image Magick thumbnails turned out on your page - I'm sure the alignment stuff is something we could take a look at fixing with your CSS grid!

Comment (Tiny-Pickles) [reply docs/web/galleryTest.html:14]: I agree with Mia, I think if you do something with CSS grid that would help fix your alignment issues. So far good work! 👍

**Review by Tiny-Pickles — APPROVED** [review]

**Comment by ebeshero** [comment]
@arrowarchive With this commit: https://github.com/newtfire/newTutorials/pull/13/commits/2673902a75112a1ca6df951419e038665d87790d , I added CSS grid to a `<section>` element wrapped around your 12 figure elements with the overlay `<figcaption>` elements. I also tinkered a bit with your positioning of the overlay. The overlay may not be where you want it (I thought it looked good at bottom right), but you should be able to adjust.  I consulted this resource on CSS grid and absolute positioning: https://webdesign.tutsplus.com/tutorials/the-quirks-of-css-grid-and-absolute-positioning--cms-31437 

This may also be helpful on centering and aligning with CSS grid: https://www.joomlashack.com/blog/tutorials/center-and-align-items-in-css-grid/

**Comment by ebeshero** [comment]
@arrowarchive I'm merging in your branch, as requested, and deleting it from the remote repo. I recommend you delete your local branch after you've done a `git pull` on the main branch.

**Pull request merged**